llvm/lib/CodeGen/GlobalISel/CombinerHelperVectorOps.cpp

```diff
@@ -160,23 +160,8 @@ bool CombinerHelper::matchExtractVectorElementWithBuildVector(
   //  %extract:_(32) = COPY %arg1(s32)
   //
   //
-  //
-  //  %bv:_(<2 x s32>) = G_BUILD_VECTOR %arg1(s32), %arg2(s32)
```

Comment:
Lost comment?

Reply:
The index is never `opaque`. It is always a constant due to the pattern.

Reply:
The case is not valid anymore.